### ISSUE TITLE
Option to clear existing tags from the local storage

### ISSUE THREAD

**Issue by kneekoo** [timbru31/amazon-tag-remover#965]
It would be nice to have an easy way to remove the tags already stored in the local storage.

**Comment by timbru31** [comment]
Is this a new thing that Amazon does? From a quick look I'm unable to identify the tag in the LocalStorage. Can you give me some more refs where it's stored?

**Comment by kneekoo** [comment]
I should've saved that, I can't replicate it at the moment. It's a bit of a story here.

From Chromium (under Linux Mint 20.3), I used DuckDuckGo to search for a text I saw on a streamer's shirt: _"Isch Not am Ma, am Risch lüt a!"_
Once I submitted that, DuckDuckGo redirected me through a series of links that made me end up on the Amazon website using their tag "duckduckgo-20".

After seeing that, I went to my local storage and searched for the tag, but I can't remember if I searched for "tag" or specifically "duckduckgo-20". I found a large array and I just deleted it. Now I disabled your extension in an attempt to reproduce this, but my Local Storage keeps staying empty. It's weird, I don't know what exactly I found in that array, but you'd think it would be created again for all the other data, right?

The only place I can find it is in the Session Storage, under the `CSM_previousURL` key, with the link of the page I last visited - the one duckduckgo redirected me to.

It looks like "a!" tells DuckDuckGo to use them as an Amazon referral for searching stuff in their search box. This is a nasty feature and I don't know how to get rid of that referral if it's not supposed to even be stored in the Local Storage. Where else can I access it on Amazon, to remove it?

I don't like the idea of complete strangers logging my shopping. Anyway, in the meantime I'll remove everything stored by Amazon in my browser, in the hope that some obfuscated link between my account and DuckDuckGo's tag will be removed as a result.
